__tests__/example_models/minimal_model/deployment.json

```diff
@@ -6,7 +6,6 @@
     "name": "Test Deployment"
   },
   "contacts": {
-    "requester": "user",
-    "secondPOC": "user"
+    "owner": [{ "kind": "user", "id": "user" }]
```

Comment:
Is there a reason for "kind" over "type"? Does type have too many connotations to language types?

Reply:
It's a keyword in both TypeScript and Mongoose, which makes it a real pain to use.  I went with `kind` as an alternative, although happy to take other suggestions.

Reply:
Thought as much. The only other name I can think of is category, but I'm happy to leave it as kind.